packages/PriceLockup/PriceLockup.jsx

```diff
@@ -152,17 +219,40 @@ const PriceLockup = ({ size, price, topText, signDirection, rateText, bottomText
     <StyledWrapperAlignment between={wrapperSpacing}>
       <Box between={size !== 'large' ? 1 : undefined}>
         {topText && <Text size={size === 'large' ? 'large' : 'small'}>{topText}</Text>}
-        <StyledPriceWrapper between={size === 'small' ? 1 : 2} inline>
-          {renderPriceValueSign()}
-          {rateText && (
-            <StyledRateText data-testid="rateText" size={size}>
-              {rateText}
-            </StyledRateText>
-          )}
-        </StyledPriceWrapper>
+        <StyledRateTextWrapper ref={containerRef}>
+          <StyledPriceWrapper ref={rateTextWrapperRef} between={size === 'small' ? 1 : 2} inline>
+            <Box between={size === 'large' ? 2 : 1} inline>
+              {signDirection === 'left' ? renderDollarSign(size) : undefined}
+              <StyledPriceValue data-testid="priceValue" size={size}>
+                {price}
+                {!bottomText &&
+                  !rateText &&
+                  renderFootnoteLinks(footnoteLinksRef, footnoteLinks, footnoteLinksInline)}
+              </StyledPriceValue>
+              {signDirection === 'right' ? renderDollarSign(size) : undefined}
```

Comment:
This can be simplified to `{signDirection === 'right' && renderDollarSign(size)}`